Formula/p/php.rb

```diff
@@ -98,25 +131,29 @@ def install
     # Prevent system pear config from inhibiting pear install
     (config_path/"pear.conf").delete if (config_path/"pear.conf").exist?
 
-    # Prevent homebrew from hardcoding path to sed shim in phpize script
-    ENV["lt_cv_path_SED"] = "sed"
-
     # Identify build provider in php -v output and phpinfo()
     ENV["PHP_BUILD_PROVIDER"] = tap.user
 
     if OS.mac?
-      ENV["SASL_CFLAGS"] = "-I#{MacOS.sdk_path_if_needed}/usr/include/sasl"
+      sdk_path = MacOS.sdk_for_formula(self).path
+      ENV["SASL_CFLAGS"] = "-I#{sdk_path}/usr/include/sasl"
       ENV["SASL_LIBS"] = "-lsasl2"
+
+      # Each extension needs a direct reference to the sdk path or it won't find the headers
+      headers_path = "=#{sdk_path}/usr"
+
+      # PHP build system incorrectly links system libraries: https://github.com/php/php-src/issues/10680
+      # Homebrew's superenv can only discard these if using realpath of SDK
+      ENV["HOMEBREW_SDKROOT"] = sdk_path.realpath
+
+      # Enable LTO as it results in higher benchmarked performance
+      ENV.append_to_cflags "-flto"
```

Comment:
LTO may be cause of external modules failing. Will experiment some more and remove if needed.